windows_service/assets/configuration/spec.yaml

```
@@ -0,0 +1,31 @@
+name: Windows Service
+
+files:
+  - name: windows_service.yaml
+    options:
+      - template: init_config
+        options:
+          - template: init_config/default
+      - template: instances
+        options:
+          - name: services
+            description: |
+              List of services to monitor e.g. Dnscache, wmiApSrv, etc.
+
+              If any service is set to `ALL`, all services registered with the SCM will be monitored.
+
+              This matches all services starting with service, as if service.* is configured.
+              For an exact match, use ^service$
+            required: true
+            value:
+              type: array
+              items:
+                type: string
```

```suggestion
                type: string
              example:
                - <SERVICE_NAME_1>
                - <SERVICE_NAME_2>
```
nit: we can keep example to help with yaml formatting